<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAFEConfiguration</class>
 <widget class="QDialog" name="DialogAFEConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AFE Configuration</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLFNSupressor">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Frequency Noise Supressor</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Digital Block</string>
   </property>
   <widget class="QCheckBox" name="checkBoxDigitalHPF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital High Pass FIlter</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxK">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>28</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>33</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>K:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>65</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>70</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximum">
     <double>6.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>OFFSET remover</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFpgaFilterEnabled">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>FPGA Filter</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>123</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital Gain</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAFEConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAFEConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
